### ISSUE TITLE
[CAIA Intake]: Authenticated Experience, Profile - notification settings, Content and IA review

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#62900]
### What does your team need support for?

- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [X] Making revisions to one or more existing VA.gov pages
- [ ] Something else

## Background
We are preparing for the migration of MHV notification preferences coming over to the VA.gov profile, and want to make sure the notification names and groups we use align with how we're talking about these things on other parts of the site. There are 3 that we need input on:

### Medical images + reports
- **Current name in MHV:** VA medical images and reports available notification
- **Proposed notification group for VA.gov:** Your health care  (displays only for users with health care)
- **Proposed name for VA.gov:** Requested medical images and reports available to download
- **Trigger:** VA medical images and reports available notification is a message triggered only after a user has requested a specific set of images in the MHV “Download my data > Medical images and reports” section (details in the description of #59721). 

### Secure messaging
- **Current name is MHV**:  "New Secure Message Notifications"
- **Proposed notification group for VA.gov**: Your health care (displays only for users with health care)
- **Proposed name for VA.gov**: New secure message available
- **Trigger:** Provider has sent new secure message (example email in the description of #59721). 

### MHV newsletter notification
**Note:** This is the only "notification" that is actually a subscription. I'm proposing we deviate from the "notify by email" label we'll use everywhere else to make that clear to users. We got questions about this from users in research. "So, will I get notified that the newsletter is available, or will it actually send me the newsletter?"

- **Current name in MHV**:  "Subscribe to Bi-Weekly Email Newsletter"
- **Proposed notification group for VA.gov**: General VA information and updates (this is a new group that would display to all users; not reflected in mock-ups)
- **Proposed name for VA.gov**: My HealtheVet bi-weekly newsletter
- **Proposed opt in label for VA.gov**: Subscribe by email
- **Trigger:** MHV team schedules a newsletter

### Will this new product be released incrementally?

- [X] Yes, we'll be launching incrementally
- [ ] No, we won't

### When do you expect to launch your product to 100% of users?

Depends on MHV to VA.gov migration

### Supporting artifacts (if any)

- [Project outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/notification-preferences/mhv-notification-preferences-migration#project-outline-myhealthevet-to-vagov-notification-preference-migration)
- [User flows](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/p/B5792AF8-87F2-496D-AE7A-DAD1448A6018/canvas), see the section called "MHV on VA.gov notification settings"
- [Example screen when a user meets criteria to see all available notifications, post-migration](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/vRLvAZ1)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59721)
- Screenshot of notifications in MHV UI: 
<img width="177" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/55992116/a8e42bb6-a2c6-4568-a195-c320b7c21555">

...

### Will this be going through the collaboration cycle?

- [X] Yes, it will be going through the collaboration cycle
- [ ] No, it will not

### When does this work need to be done?

Flexible

### Do you plan to bring this to an upcoming content office hours session?

- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [X] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

### Who's on your team?

Team name: Authenticated Experience - Profile
OCTO-DE product owner: Samara Strauss
Product manager: Travis Cahill
Designer: Liz Lantz / Pam Drouin
FE engineer: Adam Whitlock
Product/team Slack channel: #accountexp-authexp

...

### Do you have an accessibility champion on your team?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Liz Lantz

**Comment by andaleliz** [comment]
FYI I think Laura Willwerth is the person I need to talk to about this but I can't find her in GH to assign this to her. I'll reach out via Slack.

**Comment by coforma-terry** [comment]
Hey @andaleliz - assigned @laurwill  for you :)

**Comment by kristinoletmuskat** [comment]
Also tagging @sterkenburgsara for awareness as she's the new IA for the MHV space!

**Comment by andaleliz** [comment]
Decisions:

- **Medical images + reports:** Requested medical images available to download
- **Secure messaging:** New secure message available
- **MHV Newsletter:** 
  - this should go into the General VA information and updates group since it is applicable to people who aren't enrolled in VA healthcare. 
  - notification name should be My HealtheVet bi-weekly newsletter
  - channel label will be "Subscribe by email"
  
  These are all reflected in [this mock-up
](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/v/oZ92Dx/a/vRLvAZ1)

When we initially launched this product, Mikki made the recommendation ([comment here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26921#issuecomment-875867813)) that we order groups by prevalence to Veterans, which puts the "General VA..." group last (and also aligns with general IA of the site).  

@kristinoletmuskat and @sterkenburgsara do you agree with that recommendation? I want to be sure we're all on the same page before I update all the screens. Thanks!

**Comment by andaleliz** [comment]
Via [Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1691170852740159), we got alignment that the General VA information and updates group belongs at the bottom of the list.

@laurwill, I have one more item we didn't talk about - the order of the notifications in the Your Health Care group.  I'm thinking alpha makes sense:
- Health appointment reminders
- New secure message available
- Prescription shipping and tracking updates
- Requested medical images available to download

Does that make sense to you too?

**Comment by sterkenburgsara** [comment]
@andaleliz Yes, this sounds great - I agree with ordering by prevalence to Veterans. Thanks for looping me in!

**Comment by laurwill** [comment]
Hi @andaleliz ,
Documenting here my notes from slack on ordering of notification types: In medical records, we ordered the domains by priority/frequency of use based on existing data. If we have data showing a clear priority order for Veterans, I would go with that order. And if we don't, then alpha order is great.

I did have a couple other (belated) ideas for how to make the labels even more consistent on this page — [I added them to Sketch here](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/vRLvAZ1?annotation=84bb4c69-b3cf-488d-af2e-c583b2c03936). Apologies for the delayed content thoughts on this, they only occurred to me on the second look. Let me know if you want to hop on a quick sync today to chat about any of this!

**Comment by laurwill** [comment]
Thanks @andaleliz ! I did another pass in Sketch — let me know if there's anything you want another look at.

**Comment by andaleliz** [comment]
I'm good! Going to capture our decisions here and close this out:

- Changed name of notification group from Your Health Care to Health Care
- Updated name of imaging notification to "Notifications when medical images you requested are available"
- Order of notifications will be:
  - Health appointment reminders
  - New secure message available
  - Prescription shipping and tracking updates
  - Notifications when medical images you requested are available
